policy/aggregation_type.go

```diff
@@ -463,8 +463,9 @@ func (id AggregationID) Contains(aggType AggregationType) bool {
 	if !aggType.IsValid() {
 		return false
 	}
-
-	return (id[int(aggType)/64] & (1 << (uint(aggType) % 64))) > 0
+	idx := int(aggType) >> 6
```

Comment:
Can you make `6` and `0x3F` consts? e.g.,
```
const (
  aggTypeBitShift = 6
  aggTypeBitMask = 63
)
```
and use those?

Reply:
Sure, sounds good